ansible/vagrant/vagrant-ansible.yml

```diff
@@ -11,8 +11,12 @@
     - public_iface: "{{ ansible_default_ipv4.interface }}"
   tasks:
     # On stock Core OS there is no /etc/hosts.
+    - name: "Check is /etc/hosts file exists"
+      stat: path=/etc/hosts
+      register: hosts
+
     - name: "Create hosts file if it is not present"
-      file: path=/etc/hosts state=touch
+      file: path=/etc/hosts state={{ "file" if hosts.stat.exists else "touch" }}
```

Comment:
should we just use 'copy' with 'force=false' ?  and some empty src file? Maybe this is better, but I'm not in love with the conditional in state= ......